src/storage/src/source/postgres.rs

```diff
@@ -48,6 +48,12 @@ static PG_EPOCH: Lazy<SystemTime> = Lazy::new(|| UNIX_EPOCH + Duration::from_sec
 /// How often a status update message should be sent to the server
 static FEEDBACK_INTERVAL: Duration = Duration::from_secs(30);
 
+/// The amount of time we should wait after the last received message before worrying about WAL lag
+static WAL_LAG_GRACE_PERIOD: Duration = Duration::from_secs(5 * 60);
+
+/// The maximum amount of WAL lag allowed before restarting the replication process
```

Comment:
nit: maybe comment what this is in minutes (or seconds, or whatever :sweat_smile:)